### PR TITLE
Added home button to cfv.md and cfv.md files

### PR DESCRIPTION
**Pull request by AugustasV**
Maybe could be problems with link, we will see. It should work!

### CODE REVIEW

File: pages/cfs.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <h1> Continuous Call For Speakers </h1>
 
 <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScnkp-dlCXdg_W5XUV-KdgykCu1lYp87UWLS3FBRddpA0yW_w/viewform?embedded=true" width="760px" height="1600px" frameborder="0" marginheight="0" marginwidth="0" scrolling="no" style="max-width: -webkit-fill-available;">Loading...</iframe>
+
+<html>
```
Comment (filfreire) [inline pages/cfs.md]: @AugustasV i believe the `<html>...</html>` aren't required here, you can just keep the center and the the button web elements.

**Review by filfreire — CHANGES_REQUESTED** [review]
@AugustasV thank you for your contribution 👍 
Please take a look at the other comment I wrote.

File: pages/cfs.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+  <br />
+ <center><a href=".../index.html"><button type="button">Home</button></a></center>
```
Comment (filipemcarvalho) [inline pages/cfs.md]: Shouldn't this be '../index.html' instead of '.../index.html' ?

File: pages/cfv.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
 <h1> Calling for venues </h1>
 
 <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSf32AbHqIXYdiT-5rC-RJd8kwEx3XrDSO0hdBQzYp8h1du0tQ/viewform?embedded=true" width="760px" height="1600px" frameborder="0" marginheight="0" marginwidth="0" scrolling="no">Loading...</iframe>
+
+
+<html>
+  <br />
+ <center><a href=".../index.html"><button type="button">Home</button></a></center>
```
Comment (filipemcarvalho) [inline pages/cfv.md]: Same as above

**Review by AugustasV — COMMENTED** [review]
Updated files according feedback you gave .

**Review by filipemcarvalho — APPROVED** [review]

**Pull request merged**